lib/nexpose/common.rb

```diff
@@ -135,17 +135,12 @@ class Schedule < APIObject
     attr_accessor :type
     # The repeat interval based upon type.
     attr_accessor :interval
-    # The earliest date to generate the report on (in ISO 8601 format).
+    # Starting time of the scheduled scan (in ISO 8601 format).
     attr_accessor :start
-
     # The amount of time, in minutes, to allow execution before stopping.
     attr_accessor :max_duration
     # The date after which the schedule is disabled, in ISO 8601 format.
     attr_accessor :not_valid_after
-
-    # TODO: Remove this unused attribute
-    attr_accessor :incremental
```

Comment:
I think this is fine to remove without needing to bump major version. @sgreen-r7 agree?